.github/workflows/check-webroot.yml

```diff
@@ -9,6 +9,7 @@ on:
     branches:
```

Comment:
What's the reason for the changes in this file?

Reply:
Same as in `webroot-deploy-development.yml` above.